### ISSUE TITLE
OBS Game Capture is capturing incorrect viewport

### ISSUE THREAD

**Issue by Migs3** [Ryujinx/Ryujinx#3021]
### What's the issue you encountered?

Attempting to capture the image with OBS Game Capture method causes image to be flipped and capturing an incorrect .

### How can the issue be reproduced?

Launch Ryujinx, Launch OBS, Add Game Capture Method, select Ryujinx, see problem.

Here is everything as I experience it:
- The image is always flipped Vertically.  Regardless of any of the below changes.
- The area captured is affected by Graphics Scaling options and Docked/Handheld modes.
- Fullscreen vs windowed also has an effect on the captured area.
- Here are a few examples:

Again, all of the below are vertically flipped:
1. Native Scaling, Handheld mode, Windowed mode = Correct captured area at minimum window size, changing window size causes it to "shrink" in OBS.
2. Native Scaling, Handheld mode, Fullscreen mode = Capturing full pixels but it's very small size in OBS, there's a very large Black area, looks to be like 1/4 of the screen is where the game is, and 3/4 is no content/black area. (like 720p in a 1440p canvas)
3. Native Scaling, Docked mode, Windowed mode = Limited captured area, changes depending on window size
4. Native Scaling, Docked mode, Fullscreen mode = Similar to 2 above, but slightly bigger, it's like 960p image inside a 1440p canvas.
5. 2X Scaling, Docked mode, Windowed mode = Similar to 3, limited captured area. only "more zoomed"
6. 2X Scaling, Docked mode, Fullscreen mode = Limited captured area, Looks like it's capturing about 1080p of a 1440p image
7. 2X Scaling, Handheld mode, Fullscreen mode = Correct captured area
8. 2X Scaling, Handheld mode, Windowed mode = Similar to 5, limited captured area. only a little less "zoomed"

Most other games/apps capture just fine using the Game Capture method without needing to do anything, so this seems like a problem with how this is getting rendered internally.

Not expecting any kind of response/fix, just pointing out the issue for visibility and maybe helping others set theirs up to work correctly for those that don't know about it and trying to use OBS to stream/record with.

Although the fix could potentially help some incorrect rendering in the future possibly.

### Log file

N/A

### Environment?

- Ryujinx version: 1.0.7181
- Game version: Doesn't matter
- System Specs:
  - OS: Windows 10
  - CPU: Ryzen 9 5900X
  - GPU: NVidia RTX 2080
  - RAM: 32GB
- Applied Mods : No

### Additional context?

Additional info about your environment: My main screen is native 1440p, my canvas in OBS is 1080p.

**Comment by Mou-Ikkai** [comment]
Known issue, use Window Capture instead of Game Capture to fix it.

**Comment by MetrosexualGarbodor** [comment]
This was known but apparently there wasn't an issue for it, not one that I can find at least.

**Comment by Unknownforce351** [comment]
Yes, I know it was a known issue, I didn't see any actual documented issue, so I submitted it.

It is a clearly bug.  It's worth keeping track of and potentially looking into at some point, as it might be possible that it could resolve other rendering issues with some games.

I also knew of the "fix" to use window capture.  But it is not a fix.  This is a workaround and a poor one at that.  If you play in Fullscreen, OBS will not capture window content unless you click off of the window, which takes focus away from the application and thus also reduces rendering speed and causes stutters. (Standard Microsoft Windows behavior for prioritizing applications)

As I pointed out, it is possible to use Game Capture via 7 in my report, Fullscreen, Handheld, 2X Scaling as long as you flip the image in OBS.

This is a better work around in my opinion, but this also varies depending on the resolution you play at and the canvas in OBS I believe.
